confgenerator/logging_receivers.go

```diff
@@ -200,6 +200,52 @@ func init() {
 	LoggingReceiverTypes.RegisterType(func() Component { return &LoggingReceiverTCP{} })
 }
 
+// A LoggingReceiverFluentForward represents the configuration for a Forward Protocol receiver.
+type LoggingReceiverFluentForward struct {
+	ConfigComponent `yaml:",inline"`
+
+	ListenHost string `yaml:"listen_host,omitempty" validate:"omitempty,ip"`
```

Comment:
Why do we call it listen_host but validate that it's an IP and not a host?

Reply:
we had the same discussion with the tcp receiver -- it's not the best name imo but we're staying consistent with `syslog` and `tcp` which already use it for that purpose

Reply:
I think I'm not asking "can we rename it?" but "why can't we just allow hosts here too?"

Reply:
ah, for consistency, and bc that's not supported by fluent bit. we could expand it to take both in the future if we decide the work is worth it

Reply:
I just checked and it sure seems like fluent bit supports a hostname:

https://github.com/fluent/fluent-bit/blob/968cf967f938533aa4e3e0366310ef67dcf3d7df/src/flb_network.c#L1365

It's calling `getaddrinfo` on the user-supplied `listen` string, which supports both hostnames and IP addresses.

Reply:
This is a good point, I'll either add a commit to this PR for all of the `ListenHost` we have (`tcp`, `syslog` and `fluent_forward` are the only ones I see). Will also update the tests that assert on just IPs.